### PR TITLE
Jmp_Live

### PR DESCRIPTION
**Pull request by MChampagne17**
Modified scripting feature to allow new scripts for JMP Live software
## Description

Using JMP Live software script requires a PI User Name and Password to access the PI Server.  You add this in JMP LIve under assigned credentials.
## Checklist

- [x] I am legally able to add these changes to this project.
- [ ] I documentationed the changes with Natural Docs.
- [ ] I included tests using jsl-hamcrest.

Signed-off-by: Marc Champagne <marc.champagne@adm.com>

### CODE REVIEW

File: AddinFiles/addin.jmpcust
Comment (himanga) [inline AddinFiles/addin.jmpcust:1]: It looks like this file was deleted by accident, can you restore it?

File: AddinFiles/main.jsl [hunk 1]
```diff
@@ -484,12 +495,820 @@ BuildJSL = function( {}, {txt},
...
+		));
+		Return( NameExpr(NewTblExpr) );
+	);
+);
 //flag to output a bunch of status info to the log file
 debug = 0;
 
 //window box, all code is inside this box
-nw = New Window( "PI Web API Query Builder",
+nw = New Window( "PI Web API Query Builder(JMP Desktop Only)",
```
Comment (himanga) [inline AddinFiles/main.jsl:1311]: Does the JMP Desktop Only text need to stay here?

**Review by himanga — CHANGES_REQUESTED** [review]
If you add the jmpcust file back and address the couple comments this will be good to go.

File: AddinFiles/addin.jmpcust
Comment (MChampagne17) [reply AddinFiles/addin.jmpcust:1]: I see that file  in repository.

File: AddinFiles/main.jsl (hunk 1 above)
Comment (MChampagne17) [reply AddinFiles/main.jsl:1311]: I think so

**Pull request closed without merging**